soh/CMakeLists.txt

```diff
@@ -608,20 +608,20 @@ endif()
 # Pre build events
 ################################################################################
 if (MSVC)
```

Comment:
Can we switch this to check for VS generator like how I did in 2ship for this.

Reply:
What is the exact variable to check?

Reply:
On mobile right now. Just copy what 2ship has.

Reply:
Done.